docs/openapi.json

```diff
@@ -78,8 +78,8 @@
         "tags": [
           "genres"
         ],
-        "summary": "Search for a genre in the database.",
-        "description": "Search for a genre in the database.\n\nRequires: `GenreRead` permission.",
+        "summary": "Retrieve a list of genres from the database.",
+        "description": "Retrieve a list of genres from the database.\n\nRequires: `GenreRead` permission.",
```

Comment:
The change that caused this was made in the last PR but it looks like the docs were never rebuilt.

GitHub actions should be set up to do that but that's no an issue with this PR.